### PR TITLE
Include ability to delete KPI tables and data from Ona db

### PR DESCRIPTION
**Pull request by WinnyTroy**
### Changes / Features implemented
Use django ORM to remove KPI related tables and data from the Ona db

### Side effects of implementing this change

Closes onaio/infrastructure#3682

### CODE REVIEW

File: onadata/apps/main/management/commands/remove_kpi_tables.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    def handle(self, *args, **kwargs):
+        kpi_tables = [
+            'django_migrations',
+            'django_content_type',
+            'auth_user',
+            'auth_group',
+            'auth_permission',
+            'auth_group_permissions',
+            'auth_user_groups',
+            'auth_user_user_permissions',
```
Comment (WinnyTroy) [inline onadata/apps/main/management/commands/remove_kpi_tables.py:26]: Although these auth tables they were required when doing the data migration, I'm weary on droping the entire table.
There could be records referenced while dropping the other tables, but I believe we would be cascade deleting those specific records

**Comment by WinnyTroy** [comment]
> Some of these are regular Django tables. Will deleting them not break Onadata?

@moshthepitt I have highlighted afew tables I had in mind that may fail with this,
Apart from those, do you mind sharing which others you feel would break the Onadata service?

File: onadata/apps/main/management/commands/remove_kpi_tables.py (hunk 1 above)
Comment (WinnyTroy) [reply onadata/apps/main/management/commands/remove_kpi_tables.py:26]: I'll commit out the auth tables.

Comment (moshthepitt) [reply onadata/apps/main/management/commands/remove_kpi_tables.py:26]: So the way the table names work is that the app name is included e.g. `auth_user` means that this is the User model from the auth app.

You can use that scheme to only target tables from the kpi app (and any other specific app).

**Comment by WinnyTroy** [comment]
Closing this out after discussions with @ivermac on scope and implementation
This needs further discussion to introduce a policy to delete users and their resources both on Onadata and Formbuilder. This issue has been defined on the roadmap

**Pull request closed without merging**